gson/pom.xml

```diff
@@ -69,6 +69,12 @@
       <version>33.3.0-jre</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>33.2.1-jre</version>
```

Comment:
This direct dependency also fixes a different issue where previously `truth` and `guava-testlib` pulled in Guava transitively, but apparently `truth`'s Guava `-android` version was chosen by Maven.

So maybe that could have lead to issues for `guava-testlib` which expected the `-jre` version.